### ISSUE TITLE
rem vs. px usage

### ISSUE THREAD

**Issue by iteratetograceness** [jueungrace/jooniesite#40]
- "jueungraceyun" header in hero
- squiggle should be based on char size
- footer gradient height
<img width="897" alt="image" src="https://user-images.githubusercontent.com/74513600/211172747-4440ca6a-04fb-4298-a5a5-ad839378e4ea.png">
